roles/deployment/databases/tasks/postgresql.yml

```diff
@@ -14,6 +14,14 @@
 
 ---
 
+- name: Ensure psycopg2 is installed
+  package:
+    name: "{{ ((hostvars[item]['ansible_os_family'] == 'RedHat') and (hostvars[item]['ansible_distribution_major_version'] == '7')) | ternary('python-psycopg2', 'python3-psycopg2') }}"
+  with_items: "{{ databases | json_query('*.host') | unique }}"
+  delegate_to: "{{ item }}"
```

Comment:
This should be run as a task vs an inventory group. Why not modify `cloudera.cluster.infrastructure.rdbms`?

Reply:
This fix is only useful if `cloudera.cluster.infrastructure.rdbms` is not run
psycopg2 is already installed in `cloudera.cluster.infrastructure.rdbms`

Reply:
I dug into this a bit more; this application pattern is rife throughout this role, for better or worse.